src/components/images/index.js

```diff
@@ -16,7 +18,9 @@ module.exports = {
     supplyLayoutDefaults: require('./defaults'),
     includeBasePlot: require('../../plots/cartesian/include_components')('images'),
 
-    draw: require('./draw'),
+    draw: drawModule,
+    updateOnPan: drawModule,
+    updateOnPanShortCircuit: true,
```

Comment:
Not a bad solution. Nice! 

Could you just add a comment to tell future devs what this flag is about?